src/resolvers-cassandra/Settings/queries.js

```diff
@@ -6,6 +6,9 @@ const cassandraConnector = require('../../clients/cassandra/CassandraConnector')
 const withRunTime = require('../shared').withRunTime;
 const trackEvent = require('../../clients/appinsights/AppInsightsClient').trackEvent;
 
+const CONNECTOR_TWITTER = 'Twitter'
+const CONNECTOR_FACEBOOK = 'Facebook'
```

Comment:
Might as well move these constants to [shared.js](https://github.com/CatalystCode/project-fortis-services/blob/4846f58b2229a99b4ee7cd6d8bd47523313eccee/src/resolvers-cassandra/shared.js#L34) so that all the pipeline names are in one place.